### PR TITLE
Explicitly defining project folder and navigation

### PR DESCRIPTION
**Pull request by AlexJerabek**
Clarify what the application project root folder is so users run npm start in the right place.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/file-get-started-excel-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-yo-start-server.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/quickstart-yo-start-server.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809180029296540-377%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GH7QCY1RDLVCpOoZV6GWQrHs60QK%2FYsYle5UGTtxznc%3D&st=2018-09-18T00%3A25%3A43Z&se=2018-10-19T00%3A30%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks for the review @kbrandl. Updated the other quickstarts to match.

File: docs/includes/file-get-started-excel-jquery.md [hunk 1]
```diff
@@ -179,11 +179,15 @@ In this article, you'll walk through the process of building an Excel add-in by
...
 
-    
+4. Navigate to the root folder of the web application project.
+
+    ```bash
+    cd "My Office Add-in"
+    ```
 
-4. In your code editor, open **index.html** in the root of the project. This file specifies the HTML that will be rendered in the add-in's task pane. 
+5. In your code editor, open **index.html** in the root of the project. This file specifies the HTML that will be rendered in the add-in's task pane.
```
Comment (ElizabethSamuel-MSFT) [inline docs/includes/file-get-started-excel-jquery.md]: nit: do we want to make these all 1s?

**Comment by kbrandl** [comment]
@AlexJerabek thanks for updating the other quick starts as well.  Sorry for the churn on this, but can you please  rollback (undo) the changes you made to these 3 quick starts?

- [Excel - Angular quick start](https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/excel-quickstart-angular)
- [Excel - React quick start](https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/excel-quickstart-react)
- [Excel - Vue quick start](https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/excel-quickstart-vue)

Each of these quick starts needs to be significantly revised (as a result of changes made over the summer to Yo Office which render the current instructions no longer applicable). I will create a VSO work item for me to do that work, as I was already planning to revise the Angular quick start. 

Thanks again!

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/file-get-started-excel-angular.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-yo-start-server.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/quickstart-yo-start-server.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809181838301851-377%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sxXqS9Gks5kNyecR0TPS3RghH%2BSR6Zz94FGTjGWZVqc%3D&st=2018-09-18T18%3A34%3A51Z&se=2018-10-19T18%3A39%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
@kbrandl Rolled back those changes. @ElizabethSamuel-MSFT Sure, I'll get on that.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/file-get-started-excel-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-yo-start-server.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/quickstart-yo-start-server.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809181840262641-377%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DGO3GWgB%2BXiPwuW%2B%2FzmGNn60oyfaqlQz4phJiVfcziE%3D&st=2018-09-18T18%3A36%3A40Z&se=2018-10-19T18%3A41%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/file-get-started-excel-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |
[docs/includes/quickstart-yo-start-server.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Folders/docs/includes/quickstart-yo-start-server.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809181841085190-377%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=unEn92JzS%2B8tzPt3dln3X296FVUDBhGU3QsdNrOj9sM%3D&st=2018-09-18T18%3A38%3A04Z&se=2018-10-19T18%3A43%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
Looks good, thanks @AlexJerabek !

**Pull request merged**